test/fixtures/users.yml

```diff
@@ -2,4 +2,9 @@ jarjar:
   first_name: Jar Jar
   last_name: Binks
   email: jjbinks@galacticsenate.org
+
```

Comment:
We could probably consolidate it to just one test user instead of having multiple.

Reply:
tbh I prefer super generic testing names to simplify testing

Reply:
Ok, good point. In that case, could you please replace all instances of `jarjar` with `user1` for consistency, and make sure the tests don't break?